packages/sdk/client/src/services/agent-hosting-provider.ts

```diff
@@ -44,7 +76,44 @@ export class DXOSAgentHostingProviderClient implements AgentHostingProviderClien
       baseURL: runtimeAgentHostingConfig.server,
       password: this._clientConfig.get('runtime.app.env.DX_AGENTHOSTING_PASSWORD'),
     };
-    log.info('AgentHostingProviderClient initialized', { config: this._config });
+  }
+
+  init(authToken?: any) {
+    if (!this._checkAuthorization(authToken)) {
+      return false;
+    }
+
+    if (!this._config.baseURL.endsWith('/')) {
+      this._config.baseURL += '/';
+    }
+    const wsBaseURL = new URL(this.DXRPC_PATH, this._config.baseURL.replace('http', 'ws'));
+
+    this._rpc = new WebsocketRpcClient({
+      url: wsBaseURL.href,
+      requested: { AgentManager: schema.getService('dxos.service.agentmanager.AgentManager') },
+      noHandshake: true,
+    });
+
+    this._rpc.connected.on(() => {
+      this._rpcState = 'connected';
+    });
+
+    this._rpc.disconnected.on(() => {
+      this._rpcState = 'disconnected';
+    });
+    return true;
+  }
+
+  /**
+   * Check auth token from CF worker whether identity is allowed to create agent.
+   *
+   * Note: This will prevent the client from making unnecessary requests to the AgentHostingProvider API. The
```

Comment:
Prefer to put each sentence on a separate line. Makes reading and editing simpler.